src/components/TitledList.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { Text, HStack, VStack, Divider, OrderedList, ListItem, Box } from '@chakra-ui/react';
+
+interface Props {
+  title: string;
+  listElements: string[] | undefined;
+}
+
+function TitledList({ title, listElements }: Props): JSX.Element {
+  return (
+    <VStack align="stretch" spacing={4}>
+      <HStack align="center" spacing={4}>
+        <Text fontWeight="bold" fontSize="xl">
+          {title}
+        </Text>
+        <Divider width="full" borderColor="green" />
+      </HStack>
+      <Box>
+        <OrderedList listStylePos="outside" color="green" spacing={2}>
+          {listElements?.map(element => (
+            <ListItem key={element}>
+              <Box marginLeft={5}>
+                <Text as="span" color="black">
```

Comment:
I think it's fine to apply margin directly on the `Text` component, so remove the `Box` and have:
```suggestion
                <Text ml={5} textStyle="body1">
```

Reply:
This actually does not have the same effect so I'm gonna leave this one open for now.

Reply:
Oop my mistake - I think `<Text ml={5} textStyle="body1" color="black">` should work?

Reply:
The margin was only being applied to the first line when I put it on the text component.